src/plotting/plotSettings/moreOptions/moreOptionsBox.component.tsx

```diff
@@ -176,53 +374,74 @@ const MoreOptionsBox = (props: MoreOptionsProps) => {
             changeColour={changeChannelColour}
           />
         </Box>
-        <Grid container item>
-          <FormControl
-            sx={{
-              display: 'flex',
-              flexDirection: 'row',
-              width: 'inherit',
-              justifyContent: 'space-between',
-              border: 1,
-              padding: 1,
-              alignItems: 'center',
+      </Grid>
+      <Grid container item>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'row',
+            width: 'inherit',
+            justifyContent: 'space-between',
+            border: 1,
+            padding: 1,
+            alignItems: 'center',
+          }}
+        >
+          <Typography sx={{ fontSize: 12 }}>Marker colour</Typography>
```

Comment:
Not sure I'm a fan of the marker colour option - you rarely see scientific plots with marker colours different to the line colour, so this is just taking up space in the config box I feel